pkg/store/bucket.go

```diff
@@ -1352,150 +1409,141 @@ func (r *bucketIndexReader) ExpandedPostings(ms []*labels.Matcher) ([]uint64, er
 	return ps, nil
 }
 
-type postingGroup struct {
-	keys     labels.Labels
-	postings []index.Postings
+func getAllPostingsKeyLabel() labels.Label {
+	name, value := index.AllPostingsKey()
+	return labels.Label{Name: name, Value: value}
+}
 
-	aggregate func(postings []index.Postings) index.Postings
+// Logical result of each individual group is:
```

Comment:
again, struct comment should start with struct name, even not exported.

Reply:
Done